index.js

```diff
@@ -0,0 +1,123 @@
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+const session = require('express-session');
+const KnexSessionStore = require('connect-session-knex')(session);
+const bcrypt = require('bcryptjs');
+
+const db = require('./database/dbConfig.js');
+
+const server = express();
+
+
+//session cookie
+const sessionConfig = {
+  name: 'the cookie monster',
+  secret: process.env.SECRET,
+  cookie: {
+    maxAge: 1000 * 60 * 20,
```

Comment:
There is a major bug in here that set's too short lifespan of a cookie. With current age cookie will be outdated instantly and req.session.userId will be undefined.

Change that to `maxAge: 1000 * 60 * 20 * 12`